codec-http2/src/test/resources/io/netty/handler/codec/http2/testdata/testStaticTableEntries.json

```diff
@@ -62,7 +61,7 @@
         "8182 8384 8586 87 8f90",
         "9192 9394 9596 9798 999a 9b9c 9d9e 9fa0",
         "a1a2 a3a4 a5a6 a7a8 a9aa abac adae afb0",
-        "b1b2 b3b4 b5b6 b7b8 b9ba bbbc bd"
+        "b1b2 b3b4 b5b6 b7b8 babb bcbd"
```

Comment:
why the changes in this file ?

Reply:
`transfer-encoding` is a connection related header.

Reply:
@chrisvest @normanmaurer Here is the latest RFC and the changes related to headers https://datatracker.ietf.org/doc/html/rfc9113#appendix-B

Reply:
Which in turn references the list of connection headers from https://www.rfc-editor.org/rfc/rfc9110.html#name-connection